lib/plugins/transform/typeAdapter.js

```diff
@@ -120,6 +127,36 @@ module.exports = superclass => class extends superclass {
       .then(() => newValue);
   }
 
+  // object
+  objectValuesMergeAdapter(value) {
+    const newValue = {};
+    return Promise.all(Object.keys(value).map(key => super.transform(value[key])
+      .then(tmpValue => ({
+        tmpValue,
+      }))))
+      .then((xforms) => {
+        xforms.forEach(({tmpValue}) => {
+          if (!_.isEmpty(tmpValue) || _.get(this.options, 'typeAdapter.includeEmpty', true)) {
```

Comment:
Should the call to `_.get(...)` default to false if `typeAdapter.includeEmpty` is not defined?

Reply:
This matches the behavior previously defined in `objectValuesAdapter()`.

Reply:
Ok. I understand your reason for leaving it this way for now but this needs a follow up with @thebruce when he gets back.